data/i18n/hi/hi.toml

```
@@ -88,6 +94,9 @@ other = ","
 [input_placeholder_email_address]
 other = "ईमेल ऐड्रेस"
 
+[javascript_required]
+other = "इस सामग्री को देखने के लिए जावास्क्रिप्ट [सक्षम](https://www.enable-javascript.com/) होना चाहिए"
```

`````````suggestion
other = "इस सामग्री को देखने के लिए जावास्क्रिप्ट [सक्रिय](https://www.enable-javascript.com/) होना चाहिए"
```````````
This looks good here

---

Updated. Thanks.